lib/schema.ts

```diff
@@ -776,6 +851,7 @@ const RawConfigSchema = z
       })
       .optional(),
     stackSynthesizer: z.nativeEnum(stackSynthesizerType).optional(),
+    litellmConfig: LiteLLMConfig,
```

Comment:
This may not apply until you've fully migrated things but at the moment `ecsModels` is still required (it will also fail on `make deploy` due to the `modelCheck` target), if I want to just use SageMaker hosted endpoints I don't have a way to do that.

Reply:
Good call, I will send a new revision with that.

Reply:
Just tried this- this is going to be a much larger effort, I'm going to create an issue for it to follow up on it later.